src/code/bank0.asm

```diff
@@ -5157,14 +5157,14 @@ LoadTileset9::
     cp   MAP_CAVE_B                               ; $2EA2: $FE $0A
     jr   c, .label_2ED3                           ; $2EA4: $38 $2D
     ldh  a, [hMapRoom]                            ; $2EA6: $F0 $F6
-    cp   $FD                                      ; $2EA8: $FE $FD
+    cp   INDOOR_MANBO_THE_SUN_FISH_ROOM           ; $2EA8: $FE $FD
     jr   z, .label_2ED3                           ; $2EAA: $28 $27
-    cp   $B1                                      ; $2EAC: $FE $B1
+    cp   FISHING_MINIGAME_ROOM                    ; $2EAC: $FE $B1
     jr   z, .label_2ED3                           ; $2EAE: $28 $23
 
 .label_2EB0
     ld   a, [wIsBowWowFollowingLink]              ; $2EB0: $FA $56 $DB
-    cp   $01                                      ; $2EB3: $FE $01
+    cp   TRUE                                     ; $2EB3: $FE $01
```

Comment:
This is "wrong". wIsBowWowFollowingLink can have more values then 00 and 01. If I remember right it is 80 when bowwow is kidnapped.

Reply:
ok. Then I will revert that.